src/modules/getBullsAndCows.js

```diff
@@ -14,6 +14,26 @@
  */
 function getBullsAndCows(userInput, numberToGuess) {
   /* Write your code here */
+  let bulls = 0;
+  let cows = 0;
+
+  const userArray = String(userInput).split('');
+  const guessArray = String(numberToGuess).split('');
+
+  for (let i = 0; i < userArray.length; i++) {
+    if (guessArray.includes(userArray[i])) {
+      if (guessArray[i] === userArray[i]) {
+        bulls++;
+      } else {
+        cows++;
+      }
+    }
```

Comment:
The current logic checks if a digit is present in the `guessArray` and then increments cows if it's not in the correct position. However, this can lead to incorrect cow counts if a digit is counted as a bull first. Consider using a different approach to track which digits have already been matched as bulls to avoid double-counting them as cows.